### DISCUSSION TITLE
Jet Fire - FDS

### DISCUSSION THREAD

**Discussion by xaviermolina19** [firemodels/fds#13446] [Q&A] [answered]
Hi, after running the following FDS code and looking at it in PyroSim and Smokeview, I have noticed two things that I don’t quite understand.

1. My particles have an initial temperature of -42°C, and upon exiting, they come into contact with an ambient temperature of 22°C. How is it then that when looking in Smokeview, the temperature of the particles is -75.5°C, even reaching -187.1°C? (photo 1)
![photo 1-1](https://github.com/user-attachments/assets/0ac102c2-a863-4071-98cb-9a2bcee66cec)

2. As can be seen in Photo 1, many particles are "falling." Given that this is a jet fire, this should not be the case; they should exit completely horizontally. My professor mentioned that this loss is too significant. I am attaching my code in case you can help me determine what might be causing this or how it can be adjusted.

&HEAD CHID='jet_spray_P4_01', TITLE='CERTEC Jet Fire Experiments' /

&TIME T_BEGIN=-10., T_END=30./

&MISC TMPA=22, POROUS_FLOOR=F, / porous floor = F to stop FDS from removing liquid droplets from the floor of the domain, humidity based on average Barcelona RH

&MESH ID='Mesh01', IJK=100,68,20, XB=0.0,10.0,0.0,6.8,0.0,2.0, MPI_PROCESS=0/
&MESH ID='Mesh02', IJK=100,68,20, XB=0.0,10.0,0.0,6.8,2.0,4., MPI_PROCESS=1/

&VENT ID='Mesh Vent: Mesh01 [XMAX]', SURF_ID='OPEN', XB=10.0,10.0,0.0,6.8,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,0.0,6.8,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01 [YMAX]', SURF_ID='OPEN', XB=0.0,10.0,6.8,6.8,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01 [YMIN]', SURF_ID='OPEN', XB=0.0,10.0,0.0,0.0,0.0,2.0/ 
&VENT ID='Mesh Vent: Mesh01 [ZMIN]', SURF_ID='OPEN', XB=0.0,10.0,0.0,6.8,0.0,0.0/ 
&VENT ID='Mesh Vent: Mesh02 [XMAX]', SURF_ID='OPEN', XB=10.0,10.0,0.0,6.8,2.0,4.0/ 
&VENT ID='Mesh Vent: Mesh02 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,0.0,6.8,2.0,4.0/ 
&VENT ID='Mesh Vent: Mesh02 [YMAX]', SURF_ID='OPEN', XB=0.0,10.0,6.8,6.8,2.0,4.0/ 
&VENT ID='Mesh Vent: Mesh02 [YMIN]', SURF_ID='OPEN', XB=0.0,10.0,0.0,0.0,2.0,4.0/ 
&VENT ID='Mesh Vent: Mesh02 [ZMAX]', SURF_ID='OPEN', XB=0.0,10.0,0.0,6.8,4.0,4.0/  

&REAC FUEL ='PROPANE'
      SOOT_YIELD = 0.09
      CO_YIELD=0.012/ soot yield and CO yield from Zaponne, radiative fraction by default

&DEVC ID='nozzle_1', XYZ=0.5,3.4,1.15, PROP_ID='nozzle', ORIENTATION=1,0,0, QUANTITY='TIME', SETPOINT=0. / nozzle orientation is pointing to + x. nozzle activation is based on time set at 0 s.

&PART ID='propane droplets',SPEC_ID='PROPANE', QUANTITIES(1:4)='PARTICLE DIAMETER','PARTICLE TEMPERATURE','PARTICLE AGE','PARTICLE PHASE'
      DIAMETER=1000., INITIAL_TEMPERATURE=-42, SAMPLING_FACTOR=1 / 
	  
#adhere to solid -1 means droplets do not accumulate on solid surfaces, quantities (1:4) for particle visualisation
#diameter is the particle diameter in microns. plain assumption as also used in the Validation Experiment (Loughborough). Page 265 mentions it is median diameter based on operating pressure.
#initial temperature is the particle´s initial temperature 
#sampling factor means particle is to be shown in Smokeview

&PROP ID='nozzle', PART_ID='propane droplets', MASS_FLOW_RATE=0.0145,
      PARTICLE_VELOCITY=71, SPRAY_ANGLE=0.,5., PARTICLES_PER_SECOND=20000, OFFSET=0.,SMOKEVIEW_ID='nozzle'    /

&DEVC XBP=0.6,9,3.4,3.4,1.15,1.15,, QUANTITY='HRRPUV', SPATIAL_STATISTIC='VOLUME INTEGRAL', DX=0.1, DY=20., DZ=3.25, POINTS=107, Z_ID='x_HRRPUL', ID='HRRPUL', STATISTICS_START=10. /
&CTRL ID='pct', FUNCTION_TYPE='PERCENTILE', INPUT_ID='HRRPUL', PERCENTILE=0.97 /
&DEVC ID='L_F', XYZ=0,0,0, QUANTITY='CONTROL VALUE', TEMPORAL_STATISTIC='RUNNING AVERAGE', STATISTICS_START=10, STATISTICS_END=30, CTRL_ID='pct', UNITS='m' /

&DUMP DT_DEVC=5. /

&SLCF PBY=3.4, QUANTITY='TEMPERATURE', CELL_CENTERED=T /
&SLCF PBY=3.4, QUANTITY='VELOCITY', VECTOR=T /
&SLCF PBY=3.4, QUANTITY='INTEGRATED INTENSITY', CELL_CENTERED=T /
&SLCF PBZ=1.15, QUANTITY='INTEGRATED INTENSITY', CELL_CENTERED=T /
&SLCF PBY=3.4, QUANTITY='HRRPUV', CELL_CENTERED=T /
&SLCF PBY=3.4, QUANTITY='VOLUME FRACTION', SPEC_ID='PROPANE', CELL_CENTERED=T /
&SLCF PBY=3.4, QUANTITY='MACH NUMBER', CELL_CENTERED=T /

&TAIL /

**Comment by drjfloyd** [comment, marked as answer]
1.  Have you ever sprayed water on a hot day and noticed that it gets cooler?  Same thing is happening. The energy to evaporate a liquid comes from the liquid, and evaporation will continue until the liquid temperature and local vapor concentration are in equilibirum. This is basic thermodynamics.
2. You have input a 1 mm drop size. Where did you get this value from?  Is this measured or did you just make it up?  Drops are subject to the forces of gravity. Once you inject a droplet it will accelerate downwards at a rate of the force of gravity minus the force or drag opposing gravity.

**Comment by xaviermolina19** [comment]
Thank you for the response, drjfloyd. I obtained the particle diameter from the validation case in the FDS guide, the Loughborough case.

**Reply by drjfloyd** [reply]
You are not flowing anywhere near the amount of mass as in the Loughborough case.  The assumptions made for that case may not be appropriate for your case. It is your responsibilityh as the User to verify the reasonableness of any assumptions you make.

You are flowing 0.0145 kg/s which is ~ 3E-5 m3/s.  At a velocity of 71 m/s that implies a leak diameter of ~0.4 mm, much smaller than your drop size and much smaller than the 10s of mm hole sizes for Loughborough.

**Reply by xaviermolina19** [reply]
Ok, thanks for your time
